src/components/application_manager/include/application_manager/app_launch/app_launch_data_json.h

```diff
@@ -91,7 +91,8 @@ class AppLaunchDataJson : public AppLaunchDataImpl {
    * @return  pointer to json list object
    */
   Json::Value& GetApplicationListAndIndex(const ApplicationData& app_data,
-                                          int32_t& founded_index) const;
+                                          int32_t& found_index,
+                                          Json::Value& dictionary) const;
```

Comment:
@YarikMamykin missing param description

Reply:
@AKalinich-Luxoft fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/3dcb90541c1ee0310a396d2564a95eba3e12237e